src/gui/uithememanager.h

```diff
@@ -48,13 +48,20 @@ class UIThemeManager : public QObject
     QIcon getIcon(const QString &iconId) const;
     QIcon getIcon(const QString &iconId, const QString &fallback) const;
     QIcon getFlagIcon(const QString &countryIsoCode) const;
+    QPixmap getScaledPixmap(const QString &iconId, const QWidget *widget, int baseHeight) const;
+    QString getIconPath(const QString &iconId) const;
```

Comment:
Ideally it should be private. Is there any reason to keep it public?

Reply:
It's actually used to provide icon path to html

https://github.com/jagannatharjun/qBittorrent/blob/d799c7df2029cd17c134d2e66cda874f78bf8af9/src/gui/mainwindow.cpp#L1524-L1536

Reply:
https://github.com/jagannatharjun/qBittorrent/blob/d799c7df2029cd17c134d2e66cda874f78bf8af9/src/gui/mainwindow.cpp#L1524-L1536

I think the above places require hardcoded path anyway, so I will remove the changes and make `getIconPath` private.

Reply:
>I think the above places require hardcoded path anyway

Why?

Reply:
okay sorry!